src/python/espressomd/inertia_tensor.py

```diff
@@ -0,0 +1,98 @@
+import numpy as np
+
+
+def inertia_tensor(positions, masses):
+    """
+    Calculate the inertia tensor for given point masses
+    at given positions.
+
+    Parameters
+    ----------
+    positions : (N,3) array_like of :obj:`float`
+        Point masses' positions.
+    masses : (N,) array_like of :obj:`float`
+
+    Returns
+    -------
+    (3,3) array_like of :obj:`float`
+        Moment of inertia tensor.
+
+    Notes
+    -----
+    See wikipedia_.
+
+    .. _wikipedia: https://en.wikipedia.org/wiki/Moment_of_inertia#Inertia_tensor
+
+    """
+    inertia_tensor = np.zeros((3, 3))
+    for ind, p in enumerate(positions):
+        mass = masses[ind]
+        inertia_tensor[0, 0] += mass * (p[1]**2.0 + p[2]**2.0)
```

Comment:
Would't it be less error prone and verbose to write this as tensor product?

```python
inertia_tensor += mass * (np.dot(p, p) * np.eye(3) - np.outer(p, p))
```

Reply:
:+1: